service.yml

```diff
@@ -8,8 +8,7 @@ semaphore:
   pipeline_type: cp
   nano_version: true
   execution_time_limit: {"hours": 2}
-  downstream_projects: ["kafka-rest", "ksql",
-    "confluent-security-plugins", "kafka-connect-replicator",
-    "ce-kafka-rest", "confluent-cloud-plugins", "schema-registry-plugins"]
+  downstream_projects: ["kafka-rest", "ksql", "kafka-connect-replicator",
```

Comment:
Can this be something that can be optimized at the service layer. From this repo's perspective, confluent-cloud-plugins and schema-registry-plugins are dependencies. Transitive deps and optimization should be handled at the service layer in my opinion.

What if kafka-rest or ksql removed the deps later? Let try to push for them to fix in service layer if possible.